CMakeLists.txt

```diff
@@ -38,8 +38,6 @@ if ("${CMAKE_CXX_COMPILER_ID}" MATCHES "^(AppleClang|Clang|GNU)$")
   # enable those flags
   add_flag(-Wall)
   add_flag(-Wextra)
-  add_flag(-Wshadow)                 # warn the user if a variable declaration shadows one from a parent context
-  add_flag(-pedantic)                # (all versions of GCC, Clang >= 3.2) warn if non-standard C++ is used
```

Comment:
Why remove them?

Reply:
we have a lot of `shadow` errors, and they are not that scary. Example is callback inside the callback with two `ec` and `n` variables (internal definition "shadows" external)

pedantic on gcc produces warnings on ; after OUTCOME_HPP_DECLARE_ERROR